<launch>
    <arg name="mic_name" default="ReSpeaker 4 Mic Array (UAC1.0)"/>
    <arg name="api_url_list" default="{'0': 'https://ai.sweetie.bot/stt/', '1':'http://localhost:8577/'}"/>
    <node pkg="sweetie_bot_mic" type="respeaker" name="respeaker_node" output="screen">
        <rosparam param="mic_name"           subst_value="True">$(arg mic_name)</rosparam>
        <rosparam param="transcribe_servers" subst_value="True">$(arg api_url_list)</rosparam>
    </node>
</launch>
